07-memory/user_space/main.c

```diff
@@ -0,0 +1,109 @@
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+#define NVALUE 64
+#define NSTAT 10
+
+static void malloc_inv(int ind, int cal, int p1)
+{
+	clock_t tm, tf;
+	void *ptr;
+	double btm = 0;
+	double btf = 0;
+	double min_tm = 0;
+	double max_tm = 0;
+	double min_tf = 0;
+	double max_tf = 0;
```

Comment:
For results of clock() it is better to use its original type clock_t without implicit conversation.